python/whylogs/experimental/core/validators/condition_validator.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+from copy import copy
+from typing import Any, Callable, Dict, List, Optional, Union
+
+from whylogs.core.validators import ConditionValidator
+
+_validator_udfs: Dict[str, List[Dict[str, List[ConditionValidator]]]] = defaultdict(list)
+
+
+def condition_validator(
+    col_names: Union[str, List[str]],
+    condition_name: Optional[str] = None,
+    actions: List[Callable[[str, str, Any, Optional[Any]], None]] = [],
```

Comment:
```suggestion
    actions: Union[List[Callable[[str, str, Any, Optional[Any]], None]], Callable[[str, str, Any, Optional[Any]], None]]] = [],
```
Along with `actions = actions if isinstance(actions, list) else [actions]`